src/screens/DiscoveryScreenV2/hooks/useDiscovery.js

```diff
@@ -43,8 +47,8 @@ const useDiscovery = () => {
     return newTopicLists[indexTopic];
   };
 
-  const mapUser = (newUser) => {
-    return discovery.initialUsers.map((user) => {
+  const mapUserWith = (listReference, newUser) => {
+    return listReference.map((user) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `mapUserWith` is a more generalized version of the old `mapUser` function, as it now takes in a list reference to map with the new user. This change increases the flexibility and reusability of the function.



<!-- This is an auto-generated comment by OSS CodeRabbit -->